astro/src/content/blog/test-saml-configuration.mdx

```diff
@@ -0,0 +1,92 @@
+<md:AssertionConsumerService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://sptest.iamshowcase.com/acs" index="0" isDefault="true"/>
+</md:SPSSODescriptor>
+</md:EntityDescriptor>
+```
+### Configure IAMShowcase Example Application for SAML. 
+
+Select <InlineUIElement>Applications</InlineUIElement> and <InlineUIElement>Edit</InlineUIElement> the IAMShowcase Example application in FusionAuth. Select the <InlineUIElement>SAML</InlineUIElement> tab of the application and configure it with the following values.
+
+<InlineField>Enabled:</InlineField> `True`
+
+<InlineField>Issuer:</InlineField> `IAMShowcase`
+
+<InlineField>Authorized Redirect URLs:</InlineField> `https://sptest.iamshowcase.com/acs`
+
+<InlineField>Signing key:</InlineField> `Key generated for IAMShowcase example application` (This can be any key you created in Key Master) 
+
+![IAMShowcase Example SAML configuration.](/img/blogs/test-saml-configuration/fusionauth-app-saml-settings.png)
+
+### Add the user to the IAMShowcase Example application. 
+
+Select <InlineUIElement>Users</InlineUIElement>. Find the user you wish to test with. Select <InlineUIElement>Manage</InlineUIElement>. <InlineUIElement>Add registration</InlineUIElement> for the IAMShowcase Example application for the user. This user should have access to FusionAuth admin as well.
+![Showing user added to IAMShowcase Example application in FusionAuth.](/img/blogs/test-saml-configuration/fusionauth-add-user-to-application.png)
+
+### Upload the IdP metadata (for FusionAuth) to the IAMShowcase site. 
+
+Click on <InlineUIElement>Applications</InlineUIElement>. Select <InlineUIElement>View</InlineUIElement> for IAMShowcase Example. Scroll down to the <InlineField>SAML v2 integration details</InlineField> section. Copy the Metadata URL and paste that into a new browser window.
+![FusionAuth SAML integration details.](/img/blogs/test-saml-configuration/fusionauth-saml-details.png)
+
+### Update the IAMShowcase site with the Metadata.
+
+Paste the xml into the [Upload metadata](https://sptest.iamshowcase.com/instructions#start) box on the IAMShowcase site. Click Submit XML. Copy the url that pops up to your clipboard.
+
+### See it in Action.
+
+Paste the new URL into a new window in your browser. This will show information for the user that is logged into FusionAuth on the IAMShowcase website.
+![Showing user information from FusionAuth application in IAMShowcase.](/img/blogs/test-saml-configuration/iamshowcase-user-info-from-fusionauth.png)
+
+## Conclusion
+SAML might seem intimidating at first. This may not be the be-all and end-all of SAML testing, but it is a good sanity check. This will allow simplified testing. If you are having trouble, try enabling logging and you will get more information to help you troubleshoot your configuration. FusionAuth handles the heavy lifting and IAMShowcase ensures it is configured correctly.
```

Comment:
```suggestion
SAML might seem intimidating at first. Following these steps is an easy way to make sure that FusionAuth is set up correctly to work with a generic SAML SP. This allows you to test this connection before integrating FusionAuth with other SAML SPs. 

If you are having trouble, set the <InlineField>Debug Enabled</InlineField> toggle to `true` on the <Breadcrumb>SAML</Breadcrumb> tab. This outputs more information to help you troubleshoot your configuration.
```

Reply:
I'd also add where they can go to view the debug output `<Breadcrumb>Logs -> Event Log</Breadcrumb>`

Reply:
Also, re: be-all end-all, don't knock your tutorial. It is a good one.